doc/modules/grid_search.rst

```diff
@@ -1,10 +1,10 @@
-.. currentmodule:: sklearn.grid_search
+.. currentmodule:: sklearn.model_selection
 
 .. _grid_search:
 
-===============================================
-Grid Search: Searching for estimator parameters
-===============================================
+==========================================
+Search: Searching for estimator parameters
```

Comment:
We make a point of these things not having "hyper" throughout the docstrings, method names `{get,set}_params`, etc... Just saying.